### ISSUE TITLE
LabelBinarizer fails when training set is missing some labels

### ISSUE THREAD

**Issue by olliethomas** [wearepal/EthicML#605]
the label binarizer class takes a dataset with 2 labels that are not binary, and maps the min value to 0, and the max value to 1. It stores these values to do the inverse of this operation later.

There is a problem where the dataset size is very small and only contains one label due to the following:
`assert dataset.y[y_col].nunique() == 2`
I don't know the best way to deal with this, maybe provide the labels as an arg?

**Comment by thomkeh** [comment]
Yeah, just optionally providing the labels sounds like a good plan.
